### PR TITLE
Update forensic.rst for clarity and style

### PR DESCRIPTION
**Pull request by sallain**
This is a general update to the forensic disk images page. I've
clarified the rationale behind some of the processing config options
and added steps for processing compound disk images. I also cleaned up
the formatting and some of the instructions in the steps.

As a result of needing to grab links from other pages, I've also made
some small changes to appraisal.rst, dashboard-admin.rst, and
scaling-archivematica.rst, to comply with the style guidelines.

Connected to archivematica/Issues#668

### CODE REVIEW

File: user-manual/transfer/forensic.rst [hunk 1]
```diff
@@ -4,130 +4,169 @@
...
+in failures. We recommend not extracting forensic image formats. This can be set
+in the processing configuration by setting the *Extract packages* job to "No".
 
-.. figure:: images/forensic-start.*
-   :align: center
-   :figwidth: 70%
-   :width: 100%
-   :alt: Click on the report button to add metadata
+If you would like to extract forensic image formats anyway, you can set the
+*Extract packages* job to "Yes". However, we recommend testing this at scale to
```
Comment (ablwr) [inline user-manual/transfer/forensic.rst]: I think we can eliminate this second "However" in this section and just start with "We recommend testing..."

File: user-manual/transfer/forensic.rst [hunk 2]
```diff
@@ -4,130 +4,169 @@
 Forensic disk images
 ====================
 
-Workflow
---------
+Archivematica supports the preservation of forensic disk images. Selecting the
+disk image :ref:`transfer type <transfer-types>` is not required to preserve
+disk images - you can use the standard transfer type or the bag transfer types,
+if your disk image is also bagged. However, the disk image transfer type does
```
Comment (ablwr) [inline user-manual/transfer/forensic.rst]: This is nit-picky but I think we can drop the "However" and "does" here -- I feel like it makes the paragraph less 'on-the-defense' and more generally assertive.

File: user-manual/transfer/forensic.rst [hunk 3]
```diff
@@ -4,130 +4,169 @@
...
-   :width: 100%
-   :alt: Forensic disk image metadata template
+Examine contents
+++++++++++++++++
 
-   Fill in the metadata template and click save. Then navigate back to Transfer
-   to continue.
+The examine contents microservice runs `Bulk Extractor`_, a forensics tool that
+can recognize credit card numbers, social security numbers, and other patterns
+in data. The microservice creates log files that can be inspected later on.
```
Comment (ablwr) [inline user-manual/transfer/forensic.rst]: Can "The microservice creates log files that can be inspected later on." be more granular? Creates logs that get stored in the AIP / can be inspected within the Appraisal tab?

File: user-manual/transfer/forensic.rst [hunk 4]
```diff
@@ -4,130 +4,169 @@
...
 
-You may begin entering metadata for the next image loading to the dashboard by
-clicking on the Add Next button to the right of Start Transfer.
+Examine contents is another microservice that can result in increased processing
+time, especially if it is running on the contents of an extracted forensic disk
+image. If you are not extracting contents, running Bulk Extractor is
+questionably useful, as the disk image itself is not likely to generate any
```
Comment (ablwr) [inline user-manual/transfer/forensic.rst]: lol at "questionably" -- I think we can be more assertive here too and maybe cut the middle part of this paragraph, resulting in "If you are not extracting contents, the disk image itself is not likely to generate any results from Bulk Extractor."

File: user-manual/transfer/forensic.rst [hunk 5]
```diff
@@ -4,130 +4,169 @@
...
+desired, by using the backlog arrangement functionality in Archivematica.
+
+#. Start each part of the compound disk image as a single transfer, as per the
+   instructions above.
+
+#. When you reach the *Create SIP* job, select "Send to backlog". Do this for
+   each transfer.
+
+#. In the :ref:`Appraisal tab <appraisal>`, use the
+   :ref:`Arrangement <arrangement>` pane to combine your transfers into one SIP.
```
Comment (ablwr) [inline user-manual/transfer/forensic.rst:162]: You tested this, right? :grimacing: :sweat_smile: hahaha...

**Review by ablwr — COMMENTED** [review]
Just a few opinions -- feel free to disregard them!

File: user-manual/transfer/forensic.rst (hunk 5 above)
Comment (sallain) [reply user-manual/transfer/forensic.rst:162]: I did not but it... should... work? Gahhhhh.

**Comment by sallain** [comment]
@ablwr thanks! Made the recommended changes :smile_cat:

**Review by ablwr — APPROVED** [review]

**Pull request merged**